sys/net/gnrc/sock/include/gnrc_sock_internal.h

```diff
@@ -37,6 +37,22 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default port range for dynamic source port allocation
+ */
+#define GNRC_SOCK_DYN_PORTRANGE_MIN (49152U)
+#define GNRC_SOCK_DYN_PORTRANGE_MAX (65535U)
```

Comment:
the values are not, but @miri64 proposed to put these defines in this header file, hence choosing the prefix `GNRC_SOCK_` seems reasonable to me. However, if we start a naming discussion here, we can unroll every header file in RIOT and check for consistency ...